src/modules/doubleclick_mention/index.js

```diff
@@ -23,6 +23,9 @@ class DoubleClickMentionModule {
     load() {
         $(CHAT_ROOM_SELECTOR).off('dblclick.mention').on('dblclick.mention', USERNAME_SELECTORS, e => {
             if (e.shiftKey || e.ctrlKey) return;
+
+            $('.viewer-card__hide').find('button').click();
```

Comment:
i believe this is a race since there's no guarantee the viewer card hide button will be loaded by the time this callback occurs.